Leanplum-iOS-SDK.podspec

```diff
@@ -7,7 +7,8 @@ Pod::Spec.new do |s|
   s.license = { :type => 'Commercial', :text => 'See https://www.leanplum.com/tos' }
   s.author = { 'Leanplum' => 'support@leanplum.com' }
   s.social_media_url = 'https://twitter.com/leanplum'
-  s.platform = :ios, '8.0'
+  s.ios.deployment_target = '8.0'
+  s.tvos.deployment_target = '8.0'
```

Comment:
If we are changing the podspec, make sure we change it in internal one and other Cocoapods repo.